### PR TITLE
Update ko.md

### PR DESCRIPTION
**Pull request by kriskim8937**
Hi CNCF,

As a native Korean, I updated some minor flaws in this document includes :
- flaws in spacing
- changed some word to make it sounder natural.

### CODE REVIEW

**Comment by seokho-son** [comment]
As a native Korean speaker and kubernetes/website localization reviewer, 
this minor fix for Korean looks good to me.

By the way, the latest Contributor Covenant version is 2.0 (https://www.contributor-covenant.org/version/2/0/code_of_conduct/)
There is no Korean translation for v2.0 yet :)

File: code-of-conduct-languages/ko.md [hunk 1]
```diff
@@ -5,24 +5,24 @@ CNCF 커뮤니티 행동 강령(Code of Conduct) v1.0
...
 -	기타 비윤리적이거나 비전문적인 행동
 
-프로젝트 메인테이너에게는 본 행동 강령을 위반하는 코멘트, 커밋(commit), 코드, 위키(wiki) 수정, 이슈, 그리고 그 밖의 기여에 대해서 삭제, 수정, 거부할 수 있는 권한과 책임이 있습니다. 프로젝트 메인테이너는 프로젝트 관리의 모든 관점에서 이러한 행동 강령 원칙을 공정하고 일관되게 적용할 것을 약속해야 합니다. 행동 강령을 준수하지 않거나 시행하지 않는 프로젝트 메인테이너는 프로젝트 팀에서 영구적으로 제적될 수 있습니다.
+프로젝트 메인테이너에게는 본 행동 강령을 위반하는 코멘트, 커밋(commit), 코드, 위키(wiki) 수정, 이슈, 그리고 그 밖의 기여에 대해서 삭제, 수정, 거부할 수 있는 권한과 책임이 있습니다. 프로젝트 메인테이너는 프로젝트 관리의 모든 관점에서 이러한 행동 강령 원칙을 공정하고 일관되게 적용할 것을 약속해야 합니다. 행동 강령을 준수하지 않거나 시행하지 않는 프로젝트 메인테이너는 프로젝트팀에서 영구적으로 제명될 수 있습니다.
 
-본 행동 강령은 프로젝트 활동 영역 내에서 뿐만 아니라 개인이 프로젝트 또는 커뮤니티를 대변하는 공공의 활동 영역에서도 적용됩니다.
+본 행동 강령은 프로젝트 활동 영역 내에서뿐만 아니라 개인이 프로젝트 또는 커뮤니티를 대변하는 경우 공공의 활동 영역에서도 적용됩니다.
 
-쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위는 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
+쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위를 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
```
Comment (seokho-son) [inline code-of-conduct-languages/ko.md:25]: A nit, @kriskim8937 I prefer the existing expression considering the original English expression is `behavior in Kubernetes may be reported`.

```suggestion
쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위는 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
```

Comment (kriskim8937) [reply code-of-conduct-languages/ko.md]: I agree, thank you for the suggestion.

**Comment by seokho-son** [comment]
looks good to me :)

**Pull request merged**